docs/v0.0.1/search-items.md

```diff
@@ -0,0 +1,41 @@
+# Search Items
+Search view consumes a flat list of search items (in `json` format) for its contents.
+
+## Search item JSON format
+This flat list of search items takes the following form: it must contain exactly one `searchItems` root item that represents a list of search items. Each of the search items is made up of the following fields: 
+
+- `text` - The name of a given search item.
+- `iconUrl`- The absolute URL of icon of a given search item.
+- `contextData` - The string value that will be passed to the event handler when a given search item is clicked.
+- `itemType` - This value can either be `creation`, `action` or `query`. For more details of these values, please refer to [Layout Specification](./layout-specs.md) document.
+- `group` - The name of the group that a given search item belongs to.
```

Comment:
I think this will depend on what we want to show in the list mode. Cuz currently in dynamo it shows like `[group name][item type icon][category name]`